manticore/platforms/evm.py

```diff
@@ -1648,14 +1648,22 @@ def CALLDATACOPY(self, mem_offset, data_offset, size):
 
         # FIXME put zero if not enough data
         if issymbolic(size) or issymbolic(data_offset):
-            #self._constraints.add(Operators.ULE(data_offset, len(self.data)))
             self._constraints.add(Operators.ULE(size + data_offset, len(self.data) + (32 - len(self.data) % 32)))
 
         if issymbolic(size):
             raise ConcretizeStack(3, policy='ALL')
 
+        if not issymbolic(data_offset):
```

Comment:
Is data_offset able to be symbolic. If not I think it is better to just concretize it ?